tools/cockpit.spec

```diff
@@ -69,6 +69,15 @@ Patch5:         storage-btrfs.patch
 Patch101:       hide-pcp.patch
 Patch102:       0002-selinux-temporary-remove-setroubleshoot-section.patch
 
+# Leap specific patch
+
+%if 0%{?is_smo}
+# Ensure we do not affect Micro with SLE/Leap specifi patches
+%else
+%if 0%{?sle_version} >= 150400 && 0%{?sle_version} <= 150700
```

Comment:
can remove the %if here, just add patches... maybe start with number 200 for Leap, so they don't get mixed up

Reply:
we want this for both in fact

Reply:
OK, but remove the %if wrapper completely. We have the %if wrapper in the %setup around %patch